From fcd3db3988f318b73ba3d69c1435bb5525860897 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Mon, 19 Nov 2018 08:27:32 +0800
Subject: [PATCH] arm64: dts: rockchip: sdmmc clk pin driver strength to 4mA
 for rk1808

According to signal test result, sdmmc 0/1 clk pin driver strength
need to be 4mA.

Change-Id: Ifbd0515bcebe19423c4b110392c8ff8307431a23
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 5281e91084c2..1759ad965294 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -2299,7 +2299,7 @@
 
 			sdmmc0_clk: sdmmc0-clk {
 				rockchip,pins =
-					<4 RK_PA1 1 &pcfg_pull_none>;
+					<4 RK_PA1 1 &pcfg_pull_none_4ma>;
 			};
 
 			sdmmc0_detn: sdmmc0-detn {
@@ -2328,7 +2328,7 @@
 
 			sdmmc1_clk: sdmmc1-clk {
 				rockchip,pins =
-					<4 RK_PA7 1 &pcfg_pull_none>;
+					<4 RK_PA7 1 &pcfg_pull_none_4ma>;
 			};
 		};
 
-- 
2.35.3

